src/main/java/com/salesforce/WriteTopic.java

```diff
@@ -100,7 +99,7 @@ public Exception call() {
 
             // Produce one message to "warm" kafka up
             gaugeMetric(AWAITING_PRODUCE_METRIC_NAME, 1);
```

Comment:
Do we need to change the namespaces to indicate millis too?

Reply:
Maybe not this one specifically b/c it is counter

Reply:
Yeah, you probably already noticed - I changed them in BenchmarkApp.java